reference-implementation/lib/readable-stream-abstract-ops.js

```diff
@@ -117,22 +153,47 @@ export function CreateReadableStreamEnqueueFunction(stream) {
 }
 
 export function CreateReadableStreamErrorFunction(stream) {
-  return e => {
+  return e => ErrorReadableStream(stream, e);
```

Comment:
I don't think this change is necessary. We can leave it as inline (both here and in the spec).